build.xml

```diff
@@ -954,11 +988,11 @@
 
     <target name="_build_java">
         <!-- Note: we cannot use javac's 'release' option, as that does not allow accessing sun.misc.Unsafe nor
-        Nashorn's ClassFilter class as any javac modules option is invalid for relase 8. -->
-        <echo message="Compiling for Java ${target.version}..."/>
+        Nashorn's ClassFilter class as any javac modules option is invalid for release 8. -->
+        <echo message="Compiling for Java ${ant.java.version}..."/>
         <javac
                debug="true" debuglevel="${debuglevel}" encoding="utf-8"
-               destdir="${build.classes.main}" includeantruntime="false" source="${source.version}" target="${target.version}">
+               destdir="${build.classes.main}" includeantruntime="false" source="${ant.java.version}" target="${ant.java.version}">
```

Comment:
I wonder if we need to explicitly pass it, isn't it the default?

Reply:
Not sure, but I prefer us to be explicit for things. Makes it easier to reason.